common/taskana-common-data/src/main/resources/sql/monitor-data/monitor-sample-data.sql

```diff
@@ -28,54 +28,54 @@ INSERT INTO ATTACHMENT VALUES('ATT:000000000000000000000000000000000009', 'TKI:0
-INSERT INTO TASK VALUES('TKI:000000000000000000000000000000000050', 'ETI:000000000000000000000000000000000050', RELATIVE_DATE(0) , RELATIVE_DATE(0) , null             , RELATIVE_DATE(0) , RELATIVE_DATE(0) , RELATIVE_DATE(700000), 'Task50',   'teamlead-2',   'Some description.', 'Some custom Note', 1       , 'CLAIMED', 'MANUAL'                , 'L40000'          , 'CLI:000000000000000000000000000000000004', 'WBI:000000000000000000000000000000000002', 'USER-1-2'    , 'DOMAIN_C', 'BPI21'            , 'PBPI21'                  , 'John', 'MyCompany1', 'MySystem1', 'MyInstance1', 'MyType1', 'MyValue1', true   , false         , null         , 'NONE'        , null             , 'Geschaeftsstelle A'   , 'Teilkasko'           , null    , null    , null    , null    , null    , null    , null    , null     , null     , null     , null     , null     , null     , 'VALUE_50'      );
+-- TASK TABLE          (ID                                        , EXTERNAL_ID                               , CREATED          , CLAIMED          , COMPLETED        , MODIFIED         , PLANNED          , RECEIVED, DUE                  , NAME    ,   CREATOR     , DESCRIPTION          , NOTE              , PRIORITY, STATE    , CLASSIFICATION_CATEGORY , CLASSIFICATION_KEY,  CLASSIFICATION_ID                        , WORKBASKET_ID                             , WORKBASKET_KEY, DOMAIN    , BUSINESS_PROCESS_ID, PARENT_BUSINESS_PROCESS_ID, OWNER , POR_COMPANY , POR_SYSTEM , POR_INSTANCE , POR_TYPE , POR_VALUE , IS_READ, IS_TRANSFERRED, CALLBACK_INFO, callback_state, CUSTOM_ATTRIBUTES, CUSTOM_1               , CUSTOM_2              , CUSTOM_3, CUSTOM_4, CUSTOM_5, CUSTOM_6, CUSTOM_7, CUSTOM_8, CUSTOM_9,CUSTOM_10 ,CUSTOM_11 ,CUSTOM_12 ,CUSTOM_13 ,CUSTOM_14 ,CUSTOM_15 ,CUSTOM_16  );
+INSERT INTO TASK VALUES('TKI:000000000000000000000000000000000001', 'ETI:000000000000000000000000000000000001', RELATIVE_DATE(0) , RELATIVE_DATE(0) , RELATIVE_DATE(0) , RELATIVE_DATE(0) , RELATIVE_DATE(-5), null    , RELATIVE_DATE(-70000), 'Task01',   'teamlead-1',   'Some description.', 'Some custom Note', 1       , 'READY'  , 'EXTERN'                , 'L10000'          , 'CLI:000000000000000000000000000000000001', 'WBI:000000000000000000000000000000000001', 'USER-1-1'    , 'DOMAIN_C', 'BPI21'            , 'PBPI21'                  , 'John', 'MyCompany1', 'MySystem1', 'MyInstance1', 'MyType1', 'MyValue1', true   , false         , null         , 'NONE'        , null             , 'Geschaeftsstelle A'   , 'Vollkasko'           , null    , null    , null    , null    , null    , null    , null    , null     , null     , null     , null     , null     , null     , 'VALUE_01'      );
```

Comment:
as an example: this statement would fail if the database was updated with one of our update scripts. (see other comment on the sequence).